configs/sites/orion/compilers.yaml

```
@@ -43,10 +29,10 @@ compilers:
 - compiler:
     spec: oneapi@2021.2.0
     paths:
-      cc: /apps/intel-2021.2/intel-2021.2/compiler/2021.2.0/linux/bin/icx
-      cxx: /apps/intel-2021.2/intel-2021.2/compiler/2021.2.0/linux/bin/icpx
-      f77: /apps/intel-2021.2/intel-2021.2/compiler/2021.2.0/linux/bin/ifx
-      fc: /apps/intel-2021.2/intel-2021.2/compiler/2021.2.0/linux/bin/ifx
+       cc: /apps/intel-2021.2/intel-2021.2/compiler/2021.2.0/linux/bin/icx
+       cxx: /apps/intel-2021.2/intel-2021.2/compiler/2021.2.0/linux/bin/icpx
+       f77: /apps/intel-2021.2/intel-2021.2/compiler/2021.2.0/linux/bin/ifx
+       fc: /apps/intel-2021.2/intel-2021.2/compiler/2021.2.0/linux/bin/ifx
```

Indentation on spack is usually two characters.
```suggestion
      fc: /apps/intel-2021.2/intel-2021.2/compiler/2021.2.0/linux/bin/ifx
```